e2e/utils/test-utils.ts

```diff
@@ -218,21 +219,38 @@ export async function performAction(
   }
 }
 
+/**
+ * Create new workspace in All Workspaces page. Returns Workspace card and workspace name object.
+ * @param page
+ * @param options
+ */
 export async function createWorkspace(
   page: Page,
-  cdrVersionName: string = config.defaultCdrVersionName
-): Promise<WorkspaceCard> {
+  options: { cdrVersion?: string; workspaceName?: string; openDataPage?: boolean } = {}
+): Promise<[WorkspaceCard, string]> {
```

Comment:
Allows test author to specify a CDR version, workspace name or stay in Data page when create workspace is done. All parameters are optional.
Return object `[WorkspaceCard, string]` is to allow test author write lesser code in test.